Add TCB 67 about Exasol (#728)
trinodb/trino.io: _episodes/67.md

---
layout: episode
title: "67: Extra speed with Exasol and Trino"
date: 2024-10-21
tags: trino exasol query performance olap
youtube_id: "qH61QQJKYPA"
wistia_id: "update date to 24"
sections:
- time: 0:00
  title: Introduction
- time: 0:53
  title: Trino releases 461, 462, and 463
- time: 9:30
  title: Guests introductions
- time: 1:00:42
  title: Upcoming events and rounding out
introduction: |

>   Manfred is joined by Thomas Bestfleisch from the Exasol team to talk about Exasol,

Reviewer: ```suggestion
  Manfred and Cole are joined by Thomas Bestfleisch from the Exasol team to talk about Exasol,
```
:)

Author: Oopps.. thats from before I knew if you would join ;-)

* [Cole Bowden](https://www.linkedin.com/in/cole-m-bowden), Developer Advocate
  at [Firebolt](https://www.firebolt.io/)
  
## Guests

* [Thomas Bestfleisch](https://www.linkedin.com/in/thomas-bestfleisch/), 
  Senior Product Manager at [Exasol](https://www.exasol.com/)

## Releases and news

Follow are some highlights of the recent Trino releases:

[Trino 461](https://trino.io/docs/current/release/release-461.html)

* Add support for the `add_files` and `add_files_from_table` procedures in the
  Iceberg connector.

[Trino 462](https://trino.io/docs/current/release/release-462.html)

* Add support for read operations when using the Unity catalog as Iceberg REST
  catalog in the Iceberg connector.
* Improve performance and memory usage when decoding data in the CLI.

[Trino 463](https://trino.io/docs/current/release/release-463.html)

* Enable HTTP/2 for internal communication by default.
* Add `timezone()` functions.
* Include table functions with `SHOW FUNCTIONS` output.
* Add support for writing change data feed when deletion vector is enabled to
  the Delta Lake connector.

[Trino 464](https://trino.io/docs/current/release/release-464.html)

* Require JDK 23 to run Trino.

> * Add the Faker connector.

Reviewer: I will be making a reference to the League of Legends player who goes by Faker. Fair warning.

Author: All good .. I wont have a clue what you are talking about ;-)

Author: fyi @nineinchnick

* Add the Vertica connector.
* Remove the Accumulo connector.

> As usual, numerous performance improvements, bug fixes, and other features were

Reviewer: ```suggestion
As always, numerous performance improvements, bug fixes, and other features were
```
We can be even more explicit about this.

added as well.

* Trino maintainer call - great sync with some exciting news coming to the community soon. 
* Trino contributor call - [recording and minutes available now](https://github.com/trinodb/trino/wiki/Contributor-meetings#trino-contributor-call-24-oct-2024). 

> * Trino K8s operator meeting - minutes coming soon.

Reviewer: Spell out Kubernetes?

Author: 👍

> * Trino Summit call for speakers closed - stay tuned for announcements.

Reviewer: Maybe let's highlight that Trino Summit is coming soon and people should register, so there's a clearer call to action.

## Introducing Thomas and Exasol

> Exasol is a lightning fast, in-memory database for analytics. 

Reviewer: You could say it is _literally_ the fastest, given that it's been at the top of TPC-H benchmarks for like two decades. Fun thing to call out, I think.

Author: Sounds good.

<img src="{{site.baseurl}}/assets/images/logos/exasol-small.png">

## Exasol and Trino

> Trino can Exasol bridge the gap between extreme performance with in-memory usage

Reviewer: ```suggestion
Trino and Exasol bridge the gap between extreme performance with in-memory usage
```

---
layout: episode
title: "67: Extra speed with Exasol and Trino"
date: 2024-10-30
tags: trino exasol query performance olap
youtube_id: "qH61QQJKYPA"
wistia_id: "6urj9gugl9"
sections:
- time: 0:00
  title: Introduction

> - time: 0:53
>   title: Trino releases 461, 462, and 463
> - time: 9:30
>   title: Guests introductions
> - time: 1:00:42
>   title: Upcoming events and rounding out
> - time: 0:00
>   title: Introduction

Reviewer: ```suggestion
```

Reviewer: Or whatever needs fixing here...

* Trino Kubernetes operator meeting - minutes coming soon.
* Trino Summit call for speakers closed - stay tuned for announcements and
  [don't forget to register]({% post_url 2024-10-17-trino-summit-2024-tease %}).

## Introducing Thomas and Exasol

Exasol is a lightning fast, in-memory database for analytics. And this is not
just a marketing slogan. Exasol has been at the top of the TPC-H benchmarks for
a long time now. Thomas tells more about the database and his role. 
 
<img src="{{site.baseurl}}/assets/images/logos/exasol-small.png">

## Exasol and Trino

Trino and Exasol bridge the gap between extreme performance with in-memory usage
from Exasol, and massive scale from a lakehouse with Trino.

We learn more about Exasol as Thomas guides us through his [presentation about
Exasol and Trino]({{site.baseurl}}/assets/episode/tcb67-exasol.pdf), and take
the opportunity to question him for more details.

The pull request for the Exasol connector has been a long time in the works and
was finally merged for Trino 452. We talk about the motivation, the process,
the results, and the future for the connector.

## Resources

* [Exasol](https://www.exasol.com/)

> * [Trino connector](https://trino.io/docs/current/connector/exasol.html)

Reviewer: ```suggestion
* [Trino's Exasol connector](https://trino.io/docs/current/connector/exasol.html)
```
